android/ReactAndroid/build.gradle

```diff
@@ -308,10 +308,10 @@ dependencies {
     api "com.squareup.okhttp3:okhttp-urlconnection:${OKHTTP_VERSION}"
     api 'com.squareup.okio:okio:1.14.0'
     compile 'org.webkit:android-jsc:r174650'
-    implementation 'expolib_v1.com.squareup.okhttp3:okhttp:3.6.0'
-    implementation 'expolib_v1.com.squareup.okio:okio:1.13.0'
-    implementation 'expolib_v1.com.squareup.okhttp3:okhttp-urlconnection:3.6.0'
-    implementation 'expolib_v1.com.facebook.fresco:expolib_v1-imagepipeline-okhttp3:1.0.1'
+    implementation 'com.squareup.okhttp3:okhttp:3.6.0'
```

Comment:
Let's bump it to 3.10.0, which current RN uses 👍 https://github.com/facebook/react-native/blame/833954a669a980ad574b69d8d4aa4c07627171c5/ReactAndroid/gradle.properties#L15

Reply:
Dismissed to another PR